src/api/migrations/20190625141140-drop-token-table.js

```diff
@@ -1,23 +0,0 @@
-export default {
-  up: (queryInterface, Sequelize) => queryInterface.dropTable('Tokens'),
```

Comment:
@murediane why have you deleted this file? Please, no deleting or editing of migration files.